src/resources/JSON.ts

```diff
@@ -32,14 +47,24 @@ export interface EmailJSON extends ClerkResourceJSON {
 }
 
 export interface EmailAddressJSON extends ClerkResourceJSON {
-  object: 'email_address';
+  object: ObjectType.EmailAddress;
   email_address: string;
   verification: VerificationJSON | null;
   linked_to: Array<IdentificationLinkJSON>;
 }
 
+export interface FacebookAccountJSON extends ClerkResourceJSON {
+  object: ObjectType.FacebookAccount;
+  facebook_id: string;
+  approved_scopes: string;
+  email_address: string;
+  first_name: string;
+  last_name: string;
+  picture: string;
+}
```

Comment:
Talking about our backend, it would be good to merge this and the Google account one at some point in the future. Hopefully, this will be one of the outcomes of Sokratis OAuth refactoring/re-implementation.
They are awfully similar!

Reply:
True, abstracted it on the sdk side for now, JSON payload consolidation to follow.

Reply:
👍